### PR TITLE
docs: change pagination examples

### LINKED ISSUE #356: docs: fix the example for pagination component

### PR DESCRIPTION
**Pull request by AlejandroYanes**
fixes #356

### CODE REVIEW

File: src/components/Pagination/readme.md [hunk 1]
```diff
@@ -9,43 +9,64 @@ Pagination base:
...
-                <Card
-                    className="rainbow-m-bottom_small"
-                    title={title}
-                    actions={<Button variant="neutral" label="Buy" variant="outline-brand" />}>
-                    <div className="rainbow-p-around_small">{content}</div>
+            const lastItem = activePage * 3;
+            const firstItem = lastItem - 3;
+            return this.content.slice(firstItem, lastItem).map(({ title, image }) => (
+                <Card className="rainbow-m-bottom_small rainbow-m-right_small" style={{ width: '32%' }}>
+                    <div className="rainbow-flex rainbow-flex_column rainbow_vertical-stretch" style={{ height: '100%' }}>
```
Comment (LeandroTorresSicilia) [inline src/components/Pagination/readme.md]: no styles inline
fix all

File: src/components/Pagination/readme.md [hunk 2]
```diff
@@ -9,43 +9,64 @@ Pagination base:
...
+                        <img src={image} style={{ width: '100%', flex: '1' }} />
+                        <span className="rainbow-font-size-text_large rainbow-color_dark-1 rainbow-p-bottom_small rainbow-p-top_small" style={{ alignSelf: 'center' }}>{title}</span>
+                    </div>
```
Comment (TahimiLeonBravo) [inline src/components/Pagination/readme.md]: use the footer for the title

File: src/components/Pagination/readme.md [hunk 3]
```diff
@@ -9,43 +9,64 @@ Pagination base:
...
-            return (
-                <Card
-                    className="rainbow-m-bottom_small"
-                    title={title}
-                    actions={<Button variant="neutral" label="Buy" variant="outline-brand" />}>
-                    <div className="rainbow-p-around_small">{content}</div>
+            const lastItem = activePage * 3;
+            const firstItem = lastItem - 3;
+            return this.content.slice(firstItem, lastItem).map(({ title, image }) => (
+                <Card className="rainbow-m-bottom_small rainbow-m-right_small" style={{ width: '32%' }}>
```
Comment (TahimiLeonBravo) [inline src/components/Pagination/readme.md]: use the ```rainbow-m-bottom_x-large``` instead ```rainbow-m-bottom_small```

File: src/components/Pagination/readme.md [hunk 4]
```diff
@@ -58,10 +79,10 @@ Pagination base:
                 <div>
                     <GlobalHeader src="images/user/user3.jpg" />
                     <div className="rainbow-p-around_x-large rainbow-p-around_x-large">
-                        <div>
+                        <div className="rainbow-flex">
                             {this.getContent()}
                         </div>
-                        <Pagination className="rainbow-m_auto" pages={6} activePage={activePage} onChange={this.handleOnChange} />
+                        <Pagination className="rainbow-m_auto" pages={4} activePage={activePage} onChange={this.handleOnChange} />
                     </div>
```
Comment (TahimiLeonBravo) [inline src/components/Pagination/readme.md]: the arrows in the pagination componet is not centered
screen shot 2018-10-27 at 6 41 52 pm

**Pull request merged**